service/frontend/workflowHandler.go

```diff
@@ -3512,6 +3512,21 @@ func (wh *WorkflowHandler) ReapplyEvents(
 	return nil
 }
 
+// GetClusterInfo return information about cadence deployment
+func (wh *WorkflowHandler) GetClusterInfo(
+	ctx context.Context,
+) (resp *gen.ClusterInfo, err error) {
+	defer log.CapturePanic(wh.GetLogger(), &err)
+	// Count the request in the RPS, but we still accept it even if RPS is exceeded
+	wh.allow(nil)
```

Comment:
Please check the error returned by this function call.

Reply:
fixed